client/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
:mag: isn't this 11 years old?

Reply:
This file was generated by running the app after changing to typescript, so I'm inclined to stick with it. I _believe_ the intent here is that transpiling to an old version of JS helps ensure that it won't break in older browsers?